### ISSUE TITLE
[GSoD2020] Disqus Color Scheme Issue in Dark Mode

### ISSUE THREAD

**Issue by manjotsidhu** [CircuitVerse/Interactive-Book#519]
## Describe the bug
Text color for Disqus thread in dark mode should match with the color scheme.

## To Reproduce

1.  Go to 'betalearn.circuitverse.org'
2.  Click on any page, example: Binary Representation
3.  Click on Dark Mode to enable dark mode
4.  Scroll down to footer comments

## Expected behavior
Text color should be white for dark color scheme.

## Screenshots
![Untitled](https://user-images.githubusercontent.com/22657113/108806762-9ba31f80-75c8-11eb-8a91-c77ebbbce44d.png)

## Desktop (please complete the following information):

* OS: Windows 10 Pro 20H2
* Browser: Google Chrome
* Version: 85

**Comment by infern018** [comment]
Can I make a PR for resolving this issue?

**Comment by manjotsidhu** [comment]
> Can I make a PR for resolving this issue?

Sure why not

**Comment by infern018** [comment]
![image](https://user-images.githubusercontent.com/52378712/109039170-80343380-76f2-11eb-98cd-d94d92281113.png)
**disqus** is not loading at the localhost

**Comment by manjotsidhu** [comment]
> ![image](https://user-images.githubusercontent.com/52378712/109039170-80343380-76f2-11eb-98cd-d94d92281113.png)
> **disqus** is not loading at the localhost

It won't load

**Comment by manjotsidhu** [comment]
@infern018 any updates on this ? I suggest you to take a look at https://learn.circuitverse.org, Disqus color scheme is working their.

**Comment by infern018** [comment]
> @infern018 any updates on this ? I suggest you to take a look at https://learn.circuitverse.org, Disqus color scheme is working their.

actually I don't know how will I be able to see changes in my local if I do something, but i will have a look at the link you shared, how its implemented there

**Comment by manjotsidhu** [comment]
> > @infern018 any updates on this ? I suggest you to take a look at https://learn.circuitverse.org, Disqus color scheme is working their.
> 
> actually I don't know how will I be able to see changes in my local if I do something, but i will have a look at the link you shared, how its implemented there

You can make a draft PR or run netlify on your fork to check the changes.

**Comment by manjotsidhu** [comment]
> > @infern018 any updates on this ? I suggest you to take a look at https://learn.circuitverse.org, Disqus color scheme is working their.
> 
> actually I don't know how will I be able to see changes in my local if I do something, but i will have a look at the link you shared, how its implemented there

You can serve on your local server by setting up from test disqus thread.

**Comment by infern018** [comment]
hey @manjotsidhu  apologies but I am unable to run disqus on localhost

**Comment by manjotsidhu** [comment]
@infern018 you can set a global variable before iframe is loaded as `var disqus_developer = 1;`. This might work for testing use.

**Comment by infern018** [comment]
I tried that, but it's not working

**Comment by manjotsidhu** [comment]
Make sure your `disqus_config` looks like this :
```
var disqus_config = function () {
    this.disqus_developer = 1;
    this.page.url = PAGE_URL;
    this.page.identifier = PAGE_IDENTIFIER;
  };
  ```

**Comment by manjotsidhu** [comment]
Alright then, I will fix this issue. You can fix hound issues in #460

**Comment by manjotsidhu** [comment]
Closing, fixed in #528
